tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/MoneyLaunderingOperation.java

```diff
@@ -232,7 +234,7 @@ public void close(HeisenbergConnection heisenbergConnection) throws MuleExceptio
         if (getPageCalls == failOn) {
           throw new ModuleException(CONNECTIVITY, new ConnectionException("Failed to retrieve Page"));
         }
-        if (getPageCalls > 2) {
+        if (getPageCalls > 3) {
```

Comment:
We have to be careful with this, are we sure we are not impacting other tests?

Reply:
yeah checked it